x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/search_session_tour.tsx

```diff
@@ -36,13 +40,16 @@ export function useSearchSessionTour(
     if (state === SearchSessionState.Loading) {
       if (!safeHas(storage, TOUR_TAKING_TOO_LONG_STEP_KEY)) {
         timeoutHandle = window.setTimeout(() => {
+          usageCollector?.trackSessionIndicatorTourLoading();
           searchSessionIndicatorRef.openPopover();
         }, TOUR_TAKING_TOO_LONG_TIMEOUT);
       }
     }
 
     if (state === SearchSessionState.Restored) {
+      usageCollector?.trackSessionIsRestored();
       if (!safeHas(storage, TOUR_RESTORE_STEP_KEY)) {
+        usageCollector?.trackSessionIndicatorTourRestored();
```

Comment:
This call here is not reliable and can be called multiple times 😮 
Because of how this `useEffect` with `searchSessionIndicatorRef` is setup 😢 

This probably won't happen in real life, but I noticed it in new unit tests I added here: https://github.com/lukasolson/kibana/pull/18

So I put up a fix to make sure we 100% won't cause excessive tracks: https://github.com/lukasolson/kibana/pull/18